sentry-log4j2/src/test/kotlin/io/sentry/log4j2/SentryAppenderTest.kt

```diff
@@ -0,0 +1,287 @@
+package io.sentry.log4j2
+
+import com.nhaarman.mockitokotlin2.mock
+import com.nhaarman.mockitokotlin2.verify
+import io.sentry.core.SentryEvent
+import io.sentry.core.SentryLevel
+import io.sentry.core.transport.ITransport
+import java.time.Instant
+import java.time.LocalDateTime
+import java.time.ZoneId
+import kotlin.test.AfterTest
+import kotlin.test.BeforeTest
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertFalse
+import kotlin.test.assertNotNull
+import kotlin.test.assertTrue
+import org.apache.logging.log4j.Level
+import org.apache.logging.log4j.LogManager
+import org.apache.logging.log4j.Logger
+import org.apache.logging.log4j.ThreadContext
+import org.apache.logging.log4j.core.LoggerContext
+import org.apache.logging.log4j.core.config.AppenderRef
+import org.apache.logging.log4j.core.config.Configuration
+import org.apache.logging.log4j.core.config.LoggerConfig
+import org.awaitility.kotlin.await
+
+class SentryAppenderTest {
+    private class Fixture(minimumBreadcrumbLevel: Level? = null, minimumEventLevel: Level? = null) {
+        val transport = mock<ITransport>()
+        val logger: Logger
+        val loggerContext = LogManager.getContext() as LoggerContext
+
+        init {
+            loggerContext.start()
+            val config: Configuration = loggerContext.configuration
+            val appender = SentryAppender("sentry", null, "http://key@localhost/proj", minimumBreadcrumbLevel, minimumEventLevel, transport)
+            config.addAppender(appender)
+
+            val ref = AppenderRef.createAppenderRef("sentry", null, null)
+
+            val loggerConfig = LoggerConfig.createLogger(false, Level.TRACE, "sentry_logger", "true", arrayOf(ref), null, config, null)
+            loggerConfig.addAppender(appender, null, null)
+            config.addLogger(SentryAppenderTest::class.java.name, loggerConfig)
+
+            loggerContext.updateLoggers(config)
+
+            appender.start()
+            loggerContext.start()
+
+            logger = LogManager.getContext().getLogger(SentryAppenderTest::class.java.name)
+        }
+    }
+
+    private lateinit var fixture: Fixture
+
+    @AfterTest
+    fun `stop log4j2`() {
+        fixture.loggerContext.stop()
+    }
+
+    @BeforeTest
+    fun `clear MDC`() {
+        ThreadContext.clearAll()
+    }
+
+    @Test
+    fun `converts message`() {
+        fixture = Fixture(minimumEventLevel = Level.DEBUG)
+        fixture.logger.debug("testing message conversion {}, {}", 1, 2)
+
+        await.untilAsserted {
+            verify(fixture.transport).send(com.nhaarman.mockitokotlin2.check { it: SentryEvent ->
```

Comment:
I', in favor of importing the packages instead of using in the code
eg

`import com.nhaarman.mockitokotlin2.check`